integration_test/cloudbuild-integration-test.yaml

```diff
@@ -433,8 +529,10 @@ substitutions:
 # Update to correct value if adding/removing steps before adapter run.
   _ADAPTER_RUN_STEP: 'step_5'
   _ADAPTER_RUN_STEP_HTTP2: 'step_6'
+  _ADAPTER_RUN_STEP_WITH_LOCAL_BACKUP: 'step_7'
+  _ADAPTER_RUN_STEP_WITH_GCS_BACKUP: 'step_8'
   _STORE_SCP_RUN_STEP: 'step_3'
-  _COMMITMENT_SCU_STEP: 'step_18'
+  _COMMITMENT_SCU_STEP: 'step_25'
```

Comment:
Looks like there are actually only 6 new steps rather than 7, so this should be 24 rather than 25. I might have miscounted before or this file changed.